profile/README.md

```
@@ -27,28 +22,30 @@ BE : https://github.com/debate-timer/debate-timer-be
+  <img width="672" height="373" src="https://github.com/user-attachments/assets/3b2a7d68-ebea-419b-bc7d-8731167b298f" />
+  <img width="672" height="373" src="https://github.com/user-attachments/assets/58dc5d1e-a69f-4646-a7cf-0c98f7350605" />
+  <img width="672" height="373" src="https://github.com/user-attachments/assets/4593096d-d9b3-482e-8411-50091faa1dc1" />
 </p>
 
-<br><br>
 
 ## 3. 프로젝트 문화
 <p align="center">
-  <img width="672" height="373" alt="image" src="https://github.com/user-attachments/assets/b49433de-2e3e-4b72-84de-c3262721bb86" />
-  <img width="672" height="379" alt="image" src="https://github.com/user-attachments/assets/7a965282-1db0-4a97-a249-ead6bda566f4" />
+  <img width="672" height="373" src="https://github.com/user-attachments/assets/22cc3809-0e60-4630-a8ef-cc88835e953b" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)